File: Java/FizzBuzz.java
// Run:
//
//   $ javac FizzBuzz.java
//   $ java FizzBuzz
//

class ConfigEntry {
    public int modulo;
    public String message;

    public ConfigEntry(int modulo, String message) {
      this.modulo = modulo;
      this.message = message;
    }
}

public class FizzBuzz {

  public static void main(String[] args) {
    int limit = 10;
    ConfigEntry[] config = {
      new ConfigEntry(3, "Fizz"),
      new ConfigEntry(5, "Buzz"),
    };

    System.out.printf("limit %d\n", limit);
    for (ConfigEntry configEntry : config) {
      System.out.printf("config (%d, %s)\n", configEntry.modulo, configEntry.message);
    }

    for (String result : fizzbuzz(limit, config)) {
      System.out.printf("%s\n", result);
    }
  }

  public static String[] fizzbuzz(int limit, ConfigEntry[] config) {
    String[] results = { config[0].message };
    return results;
  }

}

File: Python/fizzbuzz.py
# Run:
#
#   $ python fizzbuzz.py
#

def fizzbuzz(limit, tuples):
    return [tuples[0][1]]

limit = 10
config = [(3, "Fizz"), (5, "Buzz")]

print "limit %d" % limit
print "config (%d, %s)" % config[0]
print "config (%d, %s)" % config[1]

for result in fizzbuzz(limit, config):
    print result
